libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -0,0 +1,175 @@
+/* Nav Link */
+.feds-menu-content .feds-navLink {
+  padding: 6px 32px;
+}
+
+.feds-navLink--hoverCaret[aria-expanded = "true"] + .feds-popup,
+.feds-menu-headline[aria-expanded = "true"] + .feds-menu-items {
+  display: flex;
+  flex-direction: column;
+}
+
+.feds-navLink--hoverCaret[aria-expanded = "true"]:after,
+.feds-menu-headline[aria-expanded = "true"]:after {
+  transform: rotateZ(-135deg);
+}
+
+@media (min-width: 900px) {
+  .feds-navLink--hoverCaret[aria-expanded = "true"] + .feds-popup {
+    flex-direction: row;
+  }
+}
+
+/* General menu styles */
+.feds-menu-items {
+  display: none;
+  padding: 12px 0;
+}
+
+.feds-menu-headline {
+  position: relative;
+  padding: 12px 44px 12px 32px;
+  border-bottom: solid 1px var(--feds-borderColor-menu--light);
+  color: var(--feds-color-headline--light);
+  font-weight: 600;
+  white-space: nowrap;
+  cursor: pointer;
+}
+
+[dir = "rtl"] .feds-menu-headline {
+  padding-right: 32px;
+  padding-left: 44px;
+}
+
+.feds-menu-headline:after {
+  position: absolute;
+  right: 30px;
+  top: 50%;
+  display: flex;
+  width: 6px;
+  height: 6px;
+  margin-top: -3px;
+  border-width: 0 1px 1px 0;
+  border-style: solid;
+  border-color: var(--feds-color-link--light);
+  transform-origin: 75% 75%;
+  transform: rotateZ(45deg);
+  transition: transform 0.1s ease;
+  content: "";
+}
+
+[dir = "rtl"] .feds-menu-headline:after {
+  right: unset;
+  left: 30px;
+}
+
+.feds-menu-items {
+  border-bottom: solid 1px var(--feds-borderColor-menu--light);
+}
+
+.feds-menu-items .feds-cta-wrapper {
+  padding: 6px 32px;
+}
+
+.feds-promo {
+  display: none;
+}
+
+/* Start mobile styles */
+@media (min-width: 900px) {
+  .feds-menu-content {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    max-width: var(--feds-maxWidth-nav);
+  }
+
+  .feds-menu-column {
+    padding: 12px 0;
+  }
+
+  /* Uncomment if border between simple dropdown columns are needed */
```

Comment:
Is there a jira associated with this?  Don't want it to become forgotten commented out code.